src/Components/EditProject/EditProject.tsx

```diff
@@ -60,7 +70,7 @@ updatePro(id)
    <hr className='' />
    <div className="btns d-flex justify-content-between">
```

Comment:
the buttons here are not given any  type, they will default to submit
if you start editing and changing  something, both buttons will submit and change the project details

you need to make the cancel button as type button and the other as type submit

and handle navigation when clicliking the cancel button